build-constraints.yaml

```diff
@@ -1085,17 +1085,16 @@ packages:
         - yesod-markdown
         - yesod-paginator
 
-        # Freckle packages I'm maintaining for us
+    "Freckle Engineering <freckle-engineering@renaissance.com> @freckle":
```

Comment:
Would you mind listing some github handles of maintainers here? Since @freckle is an organization no one will get pinged by it. You can add multiple handles.

Reply:
@bergmark whoops, I didn't realize that `@freckle` wouldn't work. Do teams work? We have a `@freckle/backenders` team that would be ideal to include here.

Reply:
I tried pinging `@freckle/backenders` from an external account and it didn't work :(

I'll get a few handles from my team to include.

Reply:
Right, afaik there's no way to ping teams in other orgs.

Reply:
@bergmark done! I added a few frecklers' handles.